.changeset/swift-tomatoes-care.md

```diff
@@ -0,0 +1,7 @@
+---
+'@codama/nodes-from-anchor': patch
+'@codama/renderers-js': patch
```

Comment:
Could you just remove this line from the changesets since this PR only includes test changes for this package.

Reply:
Ah yep - got that done.